components/net_traits/lib.rs

```diff
@@ -758,8 +758,10 @@ pub enum NetworkError {
     /// Could be any of the internal errors, like unsupported scheme, connection errors, etc.
     Internal(String),
     LoadCancelled,
-    /// SSL validation error that has to be handled in the HTML parser
+    /// SSL validation error, to be converted to Resource::BadCertHTML in the HTML parser
     SslValidation(String, Vec<u8>),
+    /// Crash error, to be converted to Resource::Crash in the HTML parser
```

Comment:
Nit: These docstrings are missing periods. ONly mentioned because it shows up on the doc page (if that's still working).

Reply:
fixed in c85c102311a8914f8a34f0fb6932be6b2f163382